libr/core/anal.c

```diff
@@ -1445,32 +1342,29 @@ R_API int r_core_anal_graph(RCore *core, ut64 addr, int opts) {
 			" fontname=\"%s\" fontsize=\"8\"];\n", font);
 	if (is_json)
 		r_cons_printf ("[");
-	//r_cons_flush ();
-#define inrange(x,f) ((x>=f->addr)&&(x<(f->addr+f->size)))
 	r_list_foreach (core->anal->fcns, iter, fcni) {
 		if (fcni->type & (R_ANAL_FCN_TYPE_SYM | R_ANAL_FCN_TYPE_FCN)
-				&& (addr == 0 || inrange (addr, fcni))) {
```

Comment:
Iirc macros are just to check if a function is in range. But apis are to ifentify which function is in or at given offst



> On 05 Aug 2015, at 12:43, Riccardo Schirone <notifications@github.com> wrote:
> 
> In libr/core/anal.c:
> 
> >  	r_list_foreach (core->anal->fcns, iter, fcni) {
> >  		if (fcni->type & (R_ANAL_FCN_TYPE_SYM | R_ANAL_FCN_TYPE_FCN)
> > -				&& (addr == 0 || inrange (addr, fcni))) {
> infun and inrange are the same thing. but if there are already some API functions that do that, i will replace these macros with them.
> 
> —
> Reply to this email directly or view it on GitHub.
>